src/gluonts/dataset/util.py

```diff
@@ -25,14 +25,12 @@ def to_pandas(instance: dict, freq: Optional[str] = None) -> pd.Series:
     """
     Transform a dictionary into a pandas.Series object, using its "start" and
     "target" fields.
-
     Parameters
     ----------
     instance
         Dictionary containing the time series data.
     freq
         Frequency to use in the pandas.Series index.
-
```

Comment:
Are these changes intended? Maybe we can leave them out

Reply:
unintended, removed.